content/models.py

```diff
@@ -61,3 +61,24 @@ class EthicalPost(models.Model):
 
 	def __str__(self):
 		return self.title
+
+class CustomPage(models.Model):
+	CHICAGO = "Chicago"
+	DONATIONS = "Donations"
+	FINANCIALS = "Financials"
+	WHYPERU = "WhyPeru"
+	POSTCATEGORY = (
+		(CHICAGO, 'Chicago'),
```

Comment:
hmmm I see why you're doing this but if you want to do it in a "cleaner" way, I would inherit off FlatPages and see if you could do it that way. I havent done it that way before either so you would have to do some research